docs/kubernetes-cluster-management/labels.md

```diff
@@ -0,0 +1,35 @@
+# Overview
+Labels are intended to be used to specify identifying attributes of objects that are meaningful and relevant to users, but do not directly imply semantics to the core system. Labels can be used to organise and to select subsets of objects. Labels can be attached to objects at creation time and subsequently added and modified at any time. Each object can have a set of key/value labels defined. Each Key must be unique for a given object.  
+
+In order to take full advantage of using labels, they should be applied on every resource object within a Helm chart.  ie. all deployments, services, ingresses etc.  
+
+**Note**  Deployments should have two sets of labels, one for the actual deployment and another for the pod template the deployment manages.
+
+# Labels
+Each Helm chart templated resource should have the below labels.  Example placeholders are provided for values.
+
+```
+metadata:
+  labels:
+    app: {{ quote .Values.name }}
```

Comment:
I don't think app would be 'name' in this case, as that would be the particular service in the 'app' and would most likely be something different, i.e. the application that the suite of microservices make up.